src/js/InputValidation.js

```diff
@@ -0,0 +1,23 @@
+const InputValidation = {
+  isEmptyProductInput(name, price, quantity) {
+    if (name === '' || price === '' || quantity === '') return false;
+    return true;
+  },
+
+  isUnder100Price(price) {
+    if (price < 100) return false;
+    return true;
+  },
+
+  isPriceInUnitsOf10Won(price) {
+    if (price % 10 === 0) return true;
+    return false;
+  },
+
+  isUnderMinQuantity(quantity) {
+    if (quantity < 1) return false;
+    return true;
+  },
+};
```

Comment:
```suggestion
const InputValidation = {
  isEmptyProductInput(name, price, quantity) {
    return !(name === '' || price === '' || quantity === '');
  },

  isUnder100Price(price) {
    return price >= 100;
  },

  isPriceInUnitsOf10Won(price) {
    return price % 10 === 0;
  },

  isUnderMinQuantity(quantity) {
    return quantity >= 1;
  },
};
```
이런식으로 표현할 수 있을 것 같아요!